src/bp/ui-admin/src/telemetry/telemetry_type.d.ts

```diff
@@ -0,0 +1,34 @@
+export interface StoreInfoBody {
```

Comment:
Please can you just find any other name that is more explicit than Info in all of your types ... it it just as generic as "data" so the reader like myself can't understand anything

Reply:
please nest this in the parent type as it's only used in it

Reply:
Fixed in : e135fcf